save.html

```diff
@@ -20,8 +20,23 @@ <h1>This is the Cocktail Finder saved recipes page</h1>
             <!--js rendered drink options render here-->
         </section>
         <section id="recipeRender">
+        <!-- Delete button for each recipes-->
```

Comment:
This created a random search button on the page instead of a delete button. It doesn't seem to be functional